com/win32comext/mapi/src/mapi_headers/README.md

```diff
@@ -7,10 +7,10 @@ MICROSOFT SOFTWARE LICENSE TERMS
 MICROSOFT OUTLOOK 2010 MAPI HEADER FILES
 
 These license terms are an agreement between Microsoft Corporation (or based on where you live, one of its affiliates) and you. Please read them. They apply to the software named above, which includes the media on which you received it, if any. The terms also apply to any Microsoft
-    � updates,
-    � supplements,
-    � Internet-based services, and 
-    � support services
+    • updates,

```

Comment:
these changes still have snuck in but now seem to be using an odd encoding?

Reply:
Encoding is likely utf8 given I copy-pasted back. But I can go for a `git checkout upstream/main <file>` to fully restore the file, encoding included.

Reply:
yeah, maybe the old encoding wasn't utf-8 or something, but we don't expect a diff in that section, right?